config/initializers/devise.rb

```diff
@@ -42,7 +42,7 @@
     :openid_connect,
     {
       name: :azure_ad,
-      scope: [:openid, :email, :profile],
+      scope: [:openid, :email],
       response_type: :code,
```

Comment:
"profile" scope is no longer required as we do not need to access AzureAD roles.